src/index.ts

```diff
@@ -409,6 +410,20 @@ const ceil = (x1: string, precision: number): string =>
 const round = (x1: string, precision: number): string =>
   precisionAdjust('round', x1, precision)
 
+function toBigNumberString(n: number | string): string {
```

Comment:
Change `toBns(n: number | string)`.

Add a `fixSciNotation` which does the sci-notation part and use that on all string inputs for each function that  deals with a BNS